### PR TITLE
Add more info subsection with links to forge and rustc api docs

### PR DESCRIPTION
**Pull request by spastorino**
/cc @nikomatsakis

### CODE REVIEW

File: src/more-information.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Other places to find information
+
+You are going to find [Rustc API docs site] useful and there's more supplementary documentation in [Rust Forge].
```
Comment (nikomatsakis) [inline src/more-information.md]: should we put it in a separate file? my only concern is that then it's a bit less obvious. Maybe it should just be part of `about-this-guide.md`?

Comment (spastorino) [reply src/more-information.md]: Fixed.

File: src/about-this-guide.md [hunk 2]
```diff
@@ -16,4 +16,11 @@ be found at the [GitHub repository]. If you find any mistakes in the
 guide, please file an issue about it, or even better, open a PR
 with a correction!
 
+## Other places to find information
+
+You are going to find [Rustc API docs site] useful and there's more
```
Comment (nikomatsakis) [inline src/about-this-guide.md]: Maybe this:

You might also find the following sites useful:

- [Rustc API docs] -- rustdoc documentation for the compiler
- [Forge] -- contains documentation about rust infrastructure, team procedures, and more
- [compiler-team] -- the home-base for the rust compiler team, with description of the team procedures, active working groups, and the team calendar.

Comment (spastorino) [reply src/about-this-guide.md]: :+1:

File: src/about-this-guide.md [hunk 3]
```diff
@@ -16,4 +16,15 @@ be found at the [GitHub repository]. If you find any mistakes in the
...
+## Other places to find information
+
+You might also find the following sites useful:
+
+- [Rustc API docs] -- rustdoc documentation for the compiler
+- [Forge] -- contains documentation about rust infrastructure, team procedures, and more
+- [compiler-team] -- the home-base for the rust compiler team, with description of the team procedures, active working groups, and the team calendar.
+
 [GitHub repository]: https://github.com/rust-lang/rustc-guide/
+[Rustc API docssite]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc/
```
Comment (nikomatsakis) [inline src/about-this-guide.md]:
```suggestion
[Rustc API docs]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc/
```

**Pull request merged**